Feature/v1.3.0

taqishah1214 opened the pull request:
**PR Summary**
- Angular 12 to 13 update
- Build issue fixes
- ng-bootstrap tab implementation changed as per documentation for reference https://ng-bootstrap.github.io/#/components/nav/examples

taqishah1214 committed:
Angular version 13 upgrade, package updation, gitignore updated, ng-bootsrap tabset implentation changed, resolved build error

taqishah1214 committed:
remove space and delete package-lock.json

rarestoma reviewed (changes requested):
1. The nouislider is broken (I attached image
<img width="316" alt="Screen Shot 2022-04-04 at 10 41 55 AM" src="https://user-images.githubusercontent.com/27827318/161496953-fee33a1b-09f3-47f8-9c8f-90b6ee07b58b.png">
)

2. Tabs style are broken 
    -  see on page home on the localhost and live, I also attached images
    
<img width="1148" alt="Screen Shot 2022-04-04 at 10 45 20 AM" src="https://user-images.githubusercontent.com/27827318/161497452-980cb6a3-8300-4d22-a57b-70948b99fcf5.png">

<img width="1152" alt="Screen Shot 2022-04-04 at 10 45 14 AM" src="https://user-images.githubusercontent.com/27827318/161497464-4cdb3ce4-93e9-4102-a84b-709beea4d007.png">

3. Paggination has "current" text instead of icon
<img width="394" alt="Screen Shot 2022-04-04 at 10 46 03 AM" src="https://user-images.githubusercontent.com/27827318/161497608-fec16bcc-a6a2-43c1-85d9-2c2d3c000998.png">

4. Alerts close icon not showing
<img width="1109" alt="Screen Shot 2022-04-04 at 10 46 35 AM" src="https://user-images.githubusercontent.com/27827318/161497700-abd8cb07-2942-480a-ba67-cdcb5c91aae6.png">

5. Modal when open overdrop comes over it 
<img width="1215" alt="Screen Shot 2022-04-04 at 10 48 46 AM" src="https://user-images.githubusercontent.com/27827318/161498081-bc703a4c-dd40-473c-ba4d-c1bf076086f6.png">

6. Datepicker style is broken (Live vs Localhost in images)
<img width="440" alt="Screen Shot 2022-04-04 at 10 49 26 AM" src="https://user-images.githubusercontent.com/27827318/161498267-c63daf5e-8327-4bf2-825a-c9297d738912.png">
<img width="425" alt="Screen Shot 2022-04-04 at 10 49 30 AM" src="https://user-images.githubusercontent.com/27827318/161498282-720b4dc1-4d94-4b48-aeaf-98afa79ef35d.png">

7. Carousel style broken
<img width="666" alt="Screen Shot 2022-04-04 at 10 50 20 AM" src="https://user-images.githubusercontent.com/27827318/161498386-abe66012-b61a-4a3a-a1d3-21ad7f7afe22.png">

taqishah1214 committed:
Date picker and Tab issue fixed

taqishah1214 committed:
slider issue fixed

taqishah1214 committed:
Pagination issue resolved

taqishah1214 committed:
backdrop and alert icon issue fixed

taqishah1214 committed:
css issue fixed

taqishah1214 committed:
changelog updated

rarestoma reviewed (changes requested):
@taqishah1214,

1. A single issue remained on components, the design of the carousel indicators:
<img width="1179" alt="Screen Shot 2022-04-06 at 10 44 35 AM" src="https://user-images.githubusercontent.com/27827318/161922687-7c26912f-7a23-4f00-85d8-0576c2e377af.png">

2. Also, I see that you modified the Bootstrap version to 5.1.3, we should keep it to Bootstrap 4

3. Can you please also change the typo on the landing page "A beautiful Design System for Bootstrap 4 and Angular 7. It's Free and Open Source." to "A beautiful Design System for Bootstrap 4 and Angular 13. It's Free and Open Source."

Thank you!

taqishah1214 committed:
bootstrap removed and carousel css fix

taqishah1214 commented:
1- Yesterday I added the bootstrap for ng-bootstrap according to their documentation but after adding in the CSS, our current styles override with bootstrap classes so I removed it again. Previously it wasn't there so I'm removed it.
2- Fixed
3- Fixed

taqishah1214 committed:
home component message updated for angular 13

taqishah1214 committed:
flick issue resolved
